newIDE/app/scripts/extract-extensions-document.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Launch this script to generate a list(in markdown format) of all custom extensions.
+ */
+
+const fs = require("fs");
+const https = require("https");
+
+const extensionsRegistoryURL = 'https://raw.githubusercontent.com/4ian/GDevelop-extensions/master/extensions-registry.json';
+const outputFile = 'community-made-extensions.dokuwiki.md';
+
+const writeFile = content => {
+  return new Promise((resolve, reject) => {
+    fs.writeFile(outputFile, content, err => {
+      if (err) return reject(err);
+      resolve();
+    });
+  });
+};
+
+https.get(extensionsRegistoryURL, function(res) {
```

Comment:
While it's true that your solution works, I now recommend using `axios` (as done in extract-changelog.js): 

```
const axios = require('axios');

axios.get(extensionsRegistryUrl).then(response => ...)
```

Or even better using async/await :)